packages/circuits/circuits/proveNegativeReputation.circom

```diff
@@ -0,0 +1,114 @@
+/*
+    Prove:
+        1. if user has a leaf in an existed global state tree
+        2. user state rep < 0
+        3. output all epoch keys for epoch
+*/
+
+include "../node_modules/circomlib/circuits/comparators.circom";
+include "../node_modules/circomlib/circuits/gates.circom";
+include "../node_modules/circomlib/circuits/mux1.circom";
+include "../node_modules/circomlib/circuits/poseidon.circom";
+include "../node_modules/circomlib/circuits/bitify.circom";
+include "../node_modules/circomlib/circuits/sign.circom";
+include "./sparseMerkleTree.circom";
+include "./identityCommitment.circom";
+include "./incrementalMerkleTree.circom";
+include "./modulo.circom";
```

Comment:
I think these files can be imported from `../node_modules/@unirep/circuits/circuits/***.circom`.
So we don't need the copy of `identityCommitment.circom`, ... and so on

Reply:
This causes path issues. Our unirep circuits files require iden3 files from `../../node_modules` which breaks.